app/views/properties/_property_details_top_section.html.erb

```diff
@@ -1,6 +1,13 @@
 <% content_for :page_header do %>
   <h1 class="govuk-heading-l">
     <%= property.address %>
+    <% if params[:show_raise_a_repair] %>
+    <% end %>
```

Comment:
eh? @ayghor

Reply:
yess, this is the hack to hide the button for now

Reply:
eeeeeer it's empty wat

Reply:
fixed